src/build/app/mod.rs

```diff
@@ -1448,6 +1660,41 @@ impl<'b> App<'b> {
     }
 }
 
+// Allows checking for conflicts between `Args` and `Apps` with subcommand flags
+#[cfg(debug_assertions)]
+#[derive(Debug)]
+enum Flag<'a> {
+    App(App<'a>),
+    Arg(Arg<'a>),
+}
```

Comment:
Yeah, initially that was the idea... Cloning all of our subcommands and Args made me a little sick. I ran into that because I was collecting and referencing locally owned content, just realized that's unnecessary. Sorry!